File: core/x/javaapi/codegen/src/test/resources/golden/expected/com/example/grts/Color.java
package com.example.grts;

import viaduct.java.api.reflect.Type;
import viaduct.java.api.types.GraphQLEnum;

public enum Color implements GraphQLEnum {
    RED,
    GREEN,
    BLUE;

    public static final Type<Color> Reflection = Type.ofClass(Color.class);
}

File: core/tenant/codegen/src/test/resources/golden/expected/grts/Color.kt
@file:Suppress("warnings")

package viaduct.api.grts

enum class Color : viaduct.api.types.Enum {
    RED,
    GREEN,
    BLUE;

    @OptIn(viaduct.apiannotations.InternalApi::class)
    object Reflection : viaduct.api.reflect.Type<viaduct.api.grts.Color> {
        override final val name = "Color"
        override final val kcls = viaduct.api.grts.Color::class
    }
}